packages/neo-one-node-bin/src/startNode.ts

```diff
@@ -1,27 +1,11 @@
 // tslint:disable no-object-mutation no-any
-import { FullNode, FullNodeEnvironment } from '@neo-one/node';
-import { createTest } from '@neo-one/node-neo-settings';
+import { FullNode } from '@neo-one/node';
 import { finalize } from '@neo-one/utils';
-import { of as _of } from 'rxjs';
-import { createMonitor } from './createMonitor';
-
-const createRPCEnvironment = () => ({
-  http: {
-    port: 8080,
-    host: 'localhost',
-  },
-});
-
-const createDataPath = () => './neo-one-node';
-
-const createEnvironment = (options: Partial<FullNodeEnvironment> = {}): FullNodeEnvironment => ({
-  dataPath: createDataPath(),
-  rpc: createRPCEnvironment(),
-  ...options,
-});
+import { createMonitor, getConfiguration } from './utils';
 
 export const startNode = async (): Promise<void> => {
   const monitor = createMonitor();
```

Comment:
Probably a good idea to allow configuring some of the monitor settings, like log verbosity.